samples/bluetooth/radio_coex_1wire/README.rst

```diff
@@ -0,0 +1,71 @@
+.. _bluetooth_radio_coex_1wire_sample:
+
+Bluetooth: External radio coexistence using one-wire interface
+################################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample demonstrates how to use the 1-Wire coexistence feature described in :ref:`Bluetooth® external radio coexistence <nrfxlib:bluetooth_coex>`.
```

Comment:
Wire doesn't need to be capitalized here. 

Also, the title of the page uses "one-wire", and here is "1-wire". It's better to stick to using one or the other.